docs/concepts/transactions.md

```diff
@@ -0,0 +1,51 @@
+# Transactions 
+
+## Prerequisites
+
+* [Anatomy of an SDK Application](./app-anatomy.md)
+
+## Synopsis
+
+This doc describes how transactions are defined.
+1. [Transactions](#transactions)
+2. [Transaction Definition](#transaction-definition)
+3. [Messages](#messages)
+4. [Handlers and Keepers](#handlers-and-keepers)
+5. [CLI and REST Interfaces](#cli-and-rest-interfaces)
+
+## Transactions
+
+[**Transactions**](https://github.com/cosmos/cosmos-sdk/blob/97d10210beb55ad4bd6722f7186a80bf7cb140e2/types/tx_msg.go#L36-L43)  trigger state changes. They are comprised of metadata and [messages](./modules.md#messages) that instruct modules to perform various actions.
+
+When users interact with an application and make state changes (e.g. sending coins or changing stored values), transactions are created. Transactions must be signed by the appropriate account holders as specified in the transaction definitions, then broadcast to the network. These transactions must then be included in a block, validated, and approved by the network through the consensus process. To learn about the lifecycle of a transaction, click [here](./tx-lifecycle.md).
+
+## Transaction Definition
+
+Transactions are defined by module developers and application developers. They must implement the [`Tx`](https://github.com/cosmos/cosmos-sdk/blob/73700df8c39d1fe6c3d3a1a635ac03d4bacecf55/types/tx_msg.go#L34-L41) interface and include an encoder and decoder.
+
+* **GetMsgs:** unwraps the transaction and returns a list of its message(s) - one transaction may have one or multiple messages.
+* **ValidateBasic:** includes lightweight, stateless checks on the validity of a transaction.
+* **TxEncoder:** Nodes running the consensus engine (e.g. Tendermint Core) process transactions in the `[]byte` form, so transactions are always encoded before being handled by Tendermint processes. The Cosmos SDK allows developers to specify any deterministic encoding format for their applications; the default is [`Amino`](./amino.md).
```

Comment:
process -> gossip

This paragraph needs to be reworded. Explain that the consensus engine is responsible for ordering and gossiping transaction `[]bytes`. It only knows txs in their encoded form (i.e. `[]byte`), to maintain agnosticism wrt the application and facilitate gossiping.